### ISSUE TITLE
Android app crash with custom notations

### ISSUE THREAD

**Issue by Jalson1982** [react-native-text-input-mask/react-native-text-input-mask#233]
Hello,

Android crashes with error: Could not invoke RNTextnputMask.Mask null null

Here is how I use it:

```
   <TextInputMask
      affineFormats={[]}
      affinityCalculationStrategy="WHOLE_STRING"
      customNotations={[
        {
          character: '.',
          characterSet: '-',
          isOptional: true,
        },
      ]}
      mask="[0999][.][9999]"/>
```
On IOS it works fine android no way to get it to work.

**Comment by Jalson1982** [comment]
@martintreurnicht custom notations crash both ios and android.

**Comment by Jalson1982** [comment]
@martintreurnicht so I found the issue and fixed it and if you want I can make PR. Problem is that mask and unmask also need access to customNotations as there we set Mask with the format. So you do not pass it and because there are no custom notations available it "thinks" format is invalid and the app crashes.

**Comment by martintreurnicht** [comment]
hi @Jalson1982, sorry for the late response. Yeah customNotations were never really tested, a couple of people requested it and i added in, but noted that it might have bugs. Feel free to send a PR, i have limited time to spend on this repo at the moment and will get to it eventually but the fact that there is no tests fo this repo makes it hard to validate changes at the moment. I am looking to add e2e tests in the future using Detox, but I'm not sure when i'll be able to do that exactly.

**Comment by Jalson1982** [comment]
@martintreurnicht yeah no worries. There is PR not sure is it best solution so please take a look.

**Comment by s5unanow** [comment]
Hi @martintreurnicht, could you please have a look at this bug again.
I am striving to use custom notations to get user chances to input such characters as `\:(), but for now it's crashes on Android.
